daft/daft.pyi

```diff
@@ -890,9 +890,6 @@ class PySchema:
     def _truncated_table_string(self) -> str: ...
 
 class PyExpr:
-    def _input_mapping(self) -> str | None: ...
-    def _required_columns(self) -> set[str]: ...
-    def _is_column(self) -> bool: ...
```

Comment:
This is a drive-by?

Reply:
yup! We no longer have the optimizer in python. So no need for this anymore